actions/actions.py

```diff
@@ -405,9 +405,17 @@ def validate_PERSON(
         tracker: Tracker,
         domain: Dict[Text, Any],
     ) -> Dict[Text, Any]:
-
-        if value and value.title() in tracker.get_slot("known_recipients"):
-            return {"PERSON": value.title()}
+        name = ""
+        known_recipients = tracker.get_slot("known_recipients")
+        first_names = [name.split()[0] for name in known_recipients]
+        if value:
+            name = value.title()
```

Comment:
```suggestion
        name = value.title() if value else None
        known_recipients = tracker.get_slot("known_recipients")
        first_names = [name.split()[0] for name in known_recipients]
```
just to clear up this logic

Reply:
I went for `None` instead of `''` because if there is no value it will end up as `None` anyway, and `None in [a,b,c]` still returns false. Let me know if there is other reasoning for `''` i missed!